src/components/Network/Network.vue

```diff
@@ -104,32 +132,106 @@ export default {
       ],
       ingresses_headers: [
         {
-          text: 'Name',
+          text: 'Hosts',
+          value: 'hosts',
           algin: 'start',
-          sortable: false,
+          sortable: true
+        },
+        {
+          text: 'Status',
+          value: 'status'
+        },
+        {
+          text: 'Name',
           value: 'name'
         },
+        {
+          text: 'Annotations',
+          value: 'annotations'
+        },
         {
           text: 'TLS',
           value: 'tls'
         },
         {
-          text: 'Hosts',
-          value: 'hosts'
+          text: 'Paths',
+          value: 'path'
+        },
+        {
+          text: 'Service',
+          value: 'services'
+        },
+        {
+          text: 'Created at',
+          value: 'creation_timestamp'
         }
       ]
     }
   },
-  mounted () {
+  beforeMount () {
```

Comment:
I chose mount, instead of created or beforeMount, because the user sees an empty table, which dynamically fills itself.
I thing either we keep before mounted, or we could make it async? But this change has to be consistent on **all** components like.. Container.vue, Storage.vue..

Reply:
was wip